### PR TITLE
update .github

### LINKED ISSUE #115: Update invalid link to code style page in pull request template
The current template for pr's contains a link to https://github.com/XLabsProject/iw4x-client/blob/master/CODESTYLE.md instead of https://github.com/iw4x/iw4x-client/blob/master/CODESTYLE.md

### PR DESCRIPTION
**Pull request by mxve**
- Resolves #115 
- Removed CODEOWNERS
  - From what I understand this is used to automatically assign reviewers. Using @iw4x/Developers does not work as its not public.
- Formatting fix on build.yml
- Fixed typo in README that was driving me insane

### CODE REVIEW

File: .github/workflows/build.yml
Comment (diamante0018) [inline .github/workflows/build.yml:1]: Please update everything here to use `@ v4` of github actions. using nodejs16 will throw warnings and is deprecated

**Review by diamante0018 — CHANGES_REQUESTED** [review]

**Comment by SnowyWhite** [comment]
The [SECURITY.md](https://github.com/iw4x/iw4x-client/blob/develop/SECURITY.md) file has the mail address of momo, I think this PR would be a great opportunity to fix this too!

**Comment by diamante0018** [comment]
> The [SECURITY.md](https://github.com/iw4x/iw4x-client/blob/develop/SECURITY.md) file has the mail address of momo, I think this PR would be a great opportunity to fix this too!

Should be updated by a maintainer/project owner and not via a PR although I wouldn't know.. Perhaps open an issue about this for Louvenarde to see

**Review by diamante0018 — COMMENTED** [review]
updated build file looks good to me. this should get rid of the warnings from the actions tab

**Review by Rackover — APPROVED** [review]

**Pull request merged**